common/utils/src/main/resources/error/error-conditions.json

```
@@ -3580,6 +3580,21 @@
     ],
     "sqlState" : "42805"
   },
+  "PARQUET_CONVERSION_FAILURE" : {
+    "message" : [
+      "Unable to create Parquet converter for data type <t> whose Parquet type is <parquetType>."
```

Let's assign proper names to placeholders because users might face to the text in docs.
```suggestion
      "Unable to create a Parquet converter for the data type <dataType> whose Parquet type is <parquetType>."
```